{"uid":"62600d6a54f3bf69","name":"test_get_orders_unauthorized","fullName":"tests.test_get_user_orders.TestGetUserOrders#test_get_orders_unauthorized","historyId":"92bf968573314a866ca27467231d26bc","time":{"start":1727793323313,"stop":1727793323565,"duration":252},"description":"Получение заказов без авторизации.","descriptionHtml":"<p>Получение заказов без авторизации.</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[],"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_get_user_orders"},{"name":"subSuite","value":"TestGetUserOrders"},{"name":"host","value":"SC-WS-01209"},{"name":"thread","value":"7772-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_get_user_orders"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"eabe80f36169ada2","status":"broken","statusDetails":"requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='stellarburgers.nomoreparties.site', port=443): Max retries exceeded with url: /api/orders (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001D88FA21DF0>, 'Connection to stellarburgers.nomoreparties.site timed out. (connect timeout=None)'))","time":{"start":1727792334130,"stop":1727792376251,"duration":42121}}],"categories":[],"tags":[]},"source":"62600d6a54f3bf69.json","parameterValues":[]}